src/components/data/test/DataTable-test.js

```diff
@@ -425,7 +427,7 @@ describe('DataTable', () => {
     [
       // Clicking the same column when initially ascending, transitions to descending
       {
-        initialOrder: { field: 'name', direction: 'ascending' },
+        startingOrder: { field: 'name', direction: 'ascending' },
```

Comment:
I renamed `initialOrder` to `startingOrder` to avoid confusions with the new `initialColumnsOrderDir` prop.